src/server/logging/log_format.js

```diff
@@ -110,19 +110,19 @@ export default class TransformObjStream extends Stream.Transform {
         'proc',
         'load'
       ]));
-      data.message  = ansicolors.brightBlack('memory: ');
+      data.message  = chalk.gray('memory: ');
       data.message += numeral(get(data, 'proc.mem.heapUsed')).format('0.0b');
       data.message += ' ';
-      data.message += ansicolors.brightBlack('uptime: ');
+      data.message += chalk.gray('uptime: ');
       data.message += numeral(get(data, 'proc.uptime')).format('00:00:00');
       data.message += ' ';
-      data.message += ansicolors.brightBlack('load: [');
+      data.message += chalk.gray('load: [');
```

Comment:
I've had good experience using `dim` instead of specifically choosing the color `gray`, so that it adapts to white backgrounds (like jenkins)

Reply:
I'm not sure what to do here. Chalk calls "bright black" "gray", so I am not trying to change functionality here.

Reply:
Yeah, `dim` just means lower the brightness of the default color, so whether the default color is white or black it will be gray.

Reply:
Also, if you'd rather not stop using a specific color that's fine too

Reply:
How do we know what the default color is?

Reply:
is `chalk.dim('load: [');` valid syntax?